practice/blackjack/hand.py

```diff
@@ -0,0 +1,69 @@
+from card import Card
+
+
+class Hand:
+    """Hand class, holds a list of Cards."""
+
+    def __init__(self, card_list):
+        self.card_list = card_list
+
+    def __repr__(self):
+        """
+
+        >>> Hand([Card('A', 'H'), Card('A', 'H'), Card('A', 'H')])
+        Hand([Card('A', 'H'), Card('A', 'H'), Card('A', 'H')])
+        """
+        return 'Hand({!r})'.format(
+            self.card_list
+        )
+
+    def __eq__(self, other):
+        """
+
+        >>> (Hand([Card('A', 'H'), Card('A', 'H')]) ==
+        ...  Hand([Card('A', 'H'), Card('A', 'H')]))
+        True
+        >>> (Hand([Card('A', 'H'), Card('A', 'H')]) ==
+        ...  Hand([Card('2', 'H'), Card('A', 'H')]))
+        False
+        """
+        return self.card_list == other.card_list
+
+
+def add_card_to_hand(card, hand):
+    """Adds a card to the hand.
+
+    >>> add_card_to_hand(Card('A', 'H'), Hand([Card('2', 'C'), Card('3', 'D')]))
+    Hand([Card('2', 'C'), Card('3', 'D'), Card('A', 'H')])
+    """
+    hand.card_list += [card]
+    return hand
```

Comment:
This function both modifies the hand in-place and returns that updated hand. Returning implies you don't modify in-place, so make a new `Hand` instance that has an updated card list.